x-pack/plugins/security_solution/server/lib/siem_migrations/rules/api/resources/get.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { IKibanaResponse, Logger } from '@kbn/core/server';
+import { buildRouteValidationWithZod } from '@kbn/zod-helpers';
+import {
+  GetRuleMigrationResourcesRequestParams,
+  GetRuleMigrationResourcesRequestQuery,
+  type GetRuleMigrationResourcesResponse,
+} from '../../../../../../common/siem_migrations/model/api/rules/rule_migration.gen';
+import { SIEM_RULE_MIGRATION_RESOURCES_PATH } from '../../../../../../common/siem_migrations/constants';
+import type { SecuritySolutionPluginRouter } from '../../../../../types';
+
+export const registerSiemRuleMigrationsResourceGetRoute = (
+  router: SecuritySolutionPluginRouter,
+  logger: Logger
+) => {
+  router.versioned
+    .get({
+      path: SIEM_RULE_MIGRATION_RESOURCES_PATH,
+      access: 'internal',
+      security: { authz: { requiredPrivileges: ['securitySolution'] } },
+    })
+    .addVersion(
+      {
+        version: '1',
```

Comment:
Should we use numbers or the date? I know other api's currently use a date, but not sure if there's kibana wide consistency here..